src/test/java/org/springframework/samples/petclinic/owner/OwnerTest.java

```diff
@@ -63,4 +64,36 @@ public void setPetsInternalgetPetsInternalTest() {
 		assertEquals(pet, onlyPet);
 	}
 
+	@Test
+	public void getPetTest() {	
+		Pet pet = new Pet();
+		pet.setName("Pochi");
+		instance.addPet(pet);
+		
+		assertEquals(pet, instance.getPet("Pochi"));	
+		assertEquals(pet, instance.getPet("Pochi", false)); //tests pet object exists
+		assertEquals(null, instance.getPet("Pochi", true));	//tests pet object doesn't exist
+	}
+	
+	@Test
+	public void getPetsTest() {
+		Pet pet = new Pet();
+		List<Pet> list = new ArrayList<>();
+		list.add(pet);
+		instance.addPet(pet);
+		
+		assertEquals(list, instance.getPets());
+		
+		Pet pet2 = new Pet();
+		list.add(pet2);
+		instance.addPet(pet2);
+		
+		assertEquals(list, instance.getPets());
```

Comment:
Perhaps also check that the size of `ownerInstance.getPets()` increased upon adding another pet, if possible. Also, it may be worth setting a name or something to the pet and check if the same name is found in the `ownerInstance.getPets()`

Reply:
I added a new assert checking the size of the list, but since getPets return a list of the pets, I don't know if it would be an effective test.